session.py

```diff
@@ -684,8 +684,8 @@ def get_xnat_data_address(self):
     def get_xnat_experiments_address(self):
         return self.get_xnat_data_address() + "/experiments"
 
-    def get_xnat_session_address(self, experiment_id):
-        return self.get_xnat_experiments_address() + "/" + experiment_id + "?format=csv"
+    def get_xnat_session_address(self, experiment_id, extension):
```

Comment:
My only thought is would it make sense to provide a default for `extension`?... 
It might be helpful to do something like:
```
from typing import Literal

def get_xnat_session_address(self, experiment_id: str, extension: Literal['html', 'csv', 'json'] = 'html'):
    ...
```

So the user has a better idea of what the parameters passed are.

Reply:
Also `output_format` or `format` might be a better name than `extension`.